app/views/static_pages/index.html.erb

```diff
@@ -1,2 +1,109 @@
-<h1>Home Page</h1>
+<html>
```

Comment:
`<html>` and `<body>` tags should only live inside layouts (like `application.html.erb`) as the contents of the views will get embedded inside the layout file (it replaces the `<%= yield %>` tag), and there can only be one `<html>` or `<body>` tag.  You should just be able to remove them from here and everything should still work.  👍